LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 XMTP
+Copyright (c) 2023 XMTP (xmtp.org)
```

Comment:
Same as my JS comment but I think this repo was created in 2022. So might be able to keep it 2022?